backend/service/martini/fcc.go

```diff
@@ -0,0 +1,62 @@
+package martini
+
+import (
+	"net/http"
+
+	"github.com/ekspand/trusty/api"
+	v1 "github.com/ekspand/trusty/api/v1"
+	"github.com/ekspand/trusty/pkg/fcc"
+	"github.com/go-phorce/dolly/rest"
+	"github.com/go-phorce/dolly/xhttp/httperror"
+	"github.com/go-phorce/dolly/xhttp/marshal"
+)
+
+// GetFrnHandler handles v1.PathForMartiniGetFrn
+func (s *Service) GetFrnHandler() rest.Handle {
+	return func(w http.ResponseWriter, r *http.Request, _ rest.Params) {
+		filerID := api.GetQueryString(r.URL, "filer_id")
+		if filerID == "" {
+			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("missing filer_id parameter"))
+			return
+		}
+		fccClient := fcc.NewAPIClient(s.FccBaseURL)
+		frn, err := fccClient.GetFRN(filerID)
+		if err != nil {
+			logger.Errorf("filerID=%q, err=%q", filerID, err.Error())
+			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("unknown failure. Please check server logs."))
```

Comment:
this error message is returned to client. they don't have access to server logs.
change to 
`unable to get FRN response`